internal/middleware/user.go

```diff
@@ -25,6 +25,10 @@ func GetUser(ctx *gin.Context) types.User {
 	return ctx.MustGet(userContextKey).(types.User)
 }
 
+func UserIsCacheable(user types.User) bool {
+	return user.ChosenName != "" && user.ChosenName != "Test Chosen Name"
```

Comment:
as mentioned on slack – I'll add a followup issue to not cache any user attributes

Reply:
#89